packages/plugin-core/src/web/engine/INoteRenderer.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  RenderNoteOpts,
+  RespV2,
+  RenderNotePayload,
+} from "@dendronhq/common-all";
+
+/**
+ * Extracted from DEngine
+ */
+export interface INoteRenderer {
```

Comment:
is this only applicable to the web extension? if so, should it be IWebNoteRenderer?

Reply:
The idea for this is it to not be web-specific, but common.

Reply:
> > can we take some time to refactor the preview panel web to re-use the same logic and components we have from preview panel node?
> 
> Which aspects specifically?
> 
> My approach with this refactor is to iterate by first get the web version working, test it out a bit, then migrating the local version to use the web version as a base.
> 
> If we consolidate the code now, I'm worried about causing regressions in the current preview. It'll also make this review a lot bigger.

got it, lets wait then